File: DWCS/UD6/Actividad6_2_academia_baile_POO/Persona.php
<?php
class Persona{
    private String $nombre;
    private string $apellidos;
    private int $tlf_mobil;

    public function __construct(String $nombre, String $apellidos, int $tlf_mobil){
        $this->nombre = $nombre;
        $this->apellidos = $apellidos;
        $this->tlf_mobil = $tlf_mobil;
    }

    public function verInformacion(){
        return "<p>$this->nombre $this->apellidos</p>"."<p>$this->tlf_mobil</p>";
    }

    

}

File: DWPROG/Eval2/UD2_6_7_Object_Oriented_Programming/UML-Legislador/src/Persona.java
public abstract class Persona {
    private String DNI;
    private String nombre;
    private String apellidos;
    private int edad;

    public Persona(String DNI, String nombre, String apellidos, int edad){
        this.DNI = DNI;
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.edad = edad;
    }

    public  String getDNI(){
        return DNI;
    }
    public String getNombre(){
        return  nombre;
    }
    public  String getApellidos(){
        return apellidos;
    }

    public void setDNI(String DNI){
        this.DNI = DNI;
    }

    public void setNombre(String nombre){
        this.nombre = nombre;
    }
    public void setApellidos(String apellidos){
        this.apellidos = apellidos;
    }
    public void setEdad(int edad){
        this.edad = edad;
    }
}
